functions/utils/getToken.ts

```diff
@@ -11,12 +12,30 @@ function formatTitleName(symbol: string, name: string) {
   return 'View Token on Uniswap'
 }
 
-export default async function getToken(networkName: string, tokenAddress: string | undefined, url: string) {
+const convertTokenAddress = (tokenAddress: string, networkName: string) => {
+  if (tokenAddress === 'NATIVE') {
+    switch (networkName) {
+      case Chain.Celo:
+        return '0x471EcE3750Da237f93B8E339c536989b8978a438'
+      case Chain.Polygon:
+        return '0x0000000000000000000000000000000000001010'
+      default:
+        return undefined
+    }
+  }
+  return tokenAddress
+}
+
+export default async function getToken(networkName: string, tokenAddress: string, url: string) {
```

Comment:
Is this tested?

Reply:
it's tested with `api/image/tokens/[[index]].ts` and `tokens/[[index]].ts` cloud functions